config/Dockerfiles/JenkinsfileContainerBuild

```diff
@@ -163,8 +161,13 @@ podTemplate(name: podName,
                                 script: "pwd",
                                 returnStdout: true
                             ).trim()
-                            sh "buildah run -v ${cwd}/docs/source/examples/workspace:/workdir ${containerid} -- linchpin -c /etc/linchpin.conf -vvvp PinFile.dummy.yml -w /workdir up dummy-topo"
-
+                            sh "buildah copy --chown 10001:0 ${containerid} ${cwd}/config/Dockerfiles/linchpin/bin/run-linchpin /linchpin/bin/run"
```

Comment:
why is this copied here as a separate step?  why not do it in the Jenkinsfile?

also - I think the idea of the entry_user_id script is to create the user and make sure that users primary group is 0 (root) and all files should be g+rw.  This way it doesn't matter what the uid is and you shouldn't bother with this 10001 uid..

Reply:
To test with buildah, I added the linchpin-run command. There's not a way to run buildah exec lik there is in docker. To test the container, I had to run a linchpin command instead of `tail -f /dev/null` and I couldn't see another way.

It's actually not used in the final container image. If you look further, you may see that this creates a new container when I run commit. The copy is against a container and not the image. Furthermore, the image that is pushed to the docker registry is indeed the linchpin image, and not the linchpin-test image.